plugins/modules/network_compliance_workflow_manager.py

```diff
@@ -0,0 +1,1340 @@
+    dnac_debug: "{{dnac_debug}}"
+    dnac_log_level: "{{dnac_log_level}}"
+    dnac_log: False
+    state: merged
+    config:
+      - ip_address_list: ['204.1.2.2', '204.1.2.5', '204.1.2.4']
+        run_compliance:
+          trigger_full: True
+
+- name: Run full compliance check on device(s) using Site
+  cisco.dnac.network_compliance_workflow_manager:
+    dnac_host: "{{dnac_host}}"
+    dnac_username: "{{dnac_username}}"
+    dnac_password: "{{dnac_password}}"
+    dnac_verify: "{{dnac_verify}}"
+    dnac_port: "{{dnac_port}}"
+    dnac_version: "{{dnac_version}}"
+    dnac_debug: "{{dnac_debug}}"
+    dnac_log_level: "{{dnac_log_level}}"
+    dnac_log: False
+    state: merged
+    config:
+      - site_name: "Global/USA/San Francisco/Building_1/floor_1"
+        run_compliance:
+          trigger_full: True
+
+- name: Run full compliance check on device(s) using both IP address list and Site
+  cisco.dnac.network_compliance_workflow_manager:
+    dnac_host: "{{dnac_host}}"
+    dnac_username: "{{dnac_username}}"
+    dnac_password: "{{dnac_password}}"
+    dnac_verify: "{{dnac_verify}}"
+    dnac_port: "{{dnac_port}}"
+    dnac_version: "{{dnac_version}}"
+    dnac_debug: "{{dnac_debug}}"
+    dnac_log_level: "{{dnac_log_level}}"
+    dnac_log: False
+    state: merged
+    config:
+      - ip_address_list: ['204.1.2.2', '204.1.2.5', '204.1.2.4']
+        site_name: "Global/USA/San Francisco/Building_1/floor_1"
+        run_compliance:
+          trigger_full: True
+
+- name: Run compliance check with specific categories on device(s) using IP address list
+  cisco.dnac.network_compliance_workflow_manager:
+    dnac_host: "{{dnac_host}}"
+    dnac_username: "{{dnac_username}}"
+    dnac_password: "{{dnac_password}}"
+    dnac_verify: "{{dnac_verify}}"
+    dnac_port: "{{dnac_port}}"
+    dnac_version: "{{dnac_version}}"
+    dnac_debug: "{{dnac_debug}}"
+    dnac_log_level: "{{dnac_log_level}}"
+    dnac_log: False
+    state: merged
+    config:
+      - ip_address_list: ['204.1.2.2', '204.1.2.5', '204.1.2.4']
+        run_compliance:
+          trigger_full: False
+          categories: ['INTENT', 'RUNNING_CONFIG' , 'IMAGE' , 'PSIRT']
+
+- name: Run compliance check with specific categories on device(s) using Site
+  cisco.dnac.network_compliance_workflow_manager:
+    dnac_host: "{{dnac_host}}"
+    dnac_username: "{{dnac_username}}"
+    dnac_password: "{{dnac_password}}"
+    dnac_verify: "{{dnac_verify}}"
+    dnac_port: "{{dnac_port}}"
+    dnac_version: "{{dnac_version}}"
+    dnac_debug: "{{dnac_debug}}"
+    dnac_log_level: "{{dnac_log_level}}"
+    dnac_log: False
+    state: merged
+    config:
+      - site_name: "Global/USA/San Francisco/Building_1/floor_1"
+        run_compliance:
+          trigger_full: False
+          categories: ['INTENT', 'RUNNING_CONFIG' , 'IMAGE' , 'PSIRT']
+
+- name: Run compliance check with specific categories on device(s) using both IP address list and Site
```

Comment:
What is the behavior? 

Do we run the compliance for 
a) the IP addresses ['204.1.2.2', '204.1.2.5', '204.1.2.4'] and the devices located in the site "Global/USA/San Francisco/Building_1/floor_1"? 
b) the IP addresses ['204.1.2.2', '204.1.2.5', '204.1.2.4'] only
c) the devices located in the site "Global/USA/San Francisco/Building_1/floor_1" only
d) the ip address  ['204.1.2.2', '204.1.2.5', '204.1.2.4'] in  the site "Global/USA/San Francisco/Building_1/floor_1"

Reply:
a)

Both will get combined!